### PR TITLE
BUG: fix values of show axes

### PR DESCRIPTION
**Pull request by adler-j**
* Fix corner values using actual corners of the set instead of the minimum/maximum gridpoint
* Exposed `squeeze` and `coord_vectors` in `RectPartition`

This has so far not fixed the mouseover not showing coordinates issue, which should also be fixed.

### CODE REVIEW

**Comment by adler-j** [comment]
Ready for review, both issues fixed.

**Comment by adler-j** [comment]
bump for review

File: odl/discr/partition.py [hunk 1]
```diff
@@ -501,6 +506,36 @@ def append(self, other):
         """
         return self.insert(self.ndim, other)
 
+    def squeeze(self):
+        """Return the partition with removed degenerate (length 1) dimensions.
+
+        Returns
+        -------
+        squeezed : `RectPartition`
+            The squeezed partition
```
Comment (kohr-h) [inline odl/discr/partition.py]: .

File: odl/discr/partition.py [hunk 2]
```diff
@@ -501,6 +506,36 @@ def append(self, other):
...
+        >>> p = uniform_partition([0, -1], [1, 2], (3, 1))
+        >>> p.squeeze()
+        uniform_partition(0.0, 1.0, 3)
+
+        Notes
+        -----
+        This is not equivalent to
+        ``RectPartiton(self.set.squeeze(), self.grid.squeeze())`` since the
+        definition of degenerate is different in sets and grids. This functions
+        follow the definition used by grids.
```
Comment (kohr-h) [inline odl/discr/partition.py]: Check last sentence.

File: odl/util/graphics.py [hunk 3]
```diff
@@ -62,14 +62,22 @@ def _colorbar_format(minval, maxval):
...
-        maxv = grid.coord_vectors[axis].max()
+        minv = min_corner[axis]
+        maxv = max_corner[axis]
 
         points = np.linspace(minv, maxv, npoints)
         indices = np.linspace(0, grid.shape[axis] - 1, npoints, dtype=int)
         tick_values = grid.coord_vectors[axis][indices]
 
+        print(points)
+        print(tick_values)
```
Comment (kohr-h) [inline odl/util/graphics.py]: debugging stuff

**Comment by kohr-h** [comment]
Minor things, otherwise LGTM

**Pull request merged**